crates/rustfix/tests/parse_and_replace.rs

```diff
@@ -20,6 +42,7 @@ mod settings {
     pub const CHECK_JSON: &str = "RUSTFIX_TEST_CHECK_JSON";
     pub const RECORD_JSON: &str = "RUSTFIX_TEST_RECORD_JSON";
     pub const RECORD_FIXED_RUST: &str = "RUSTFIX_TEST_RECORD_FIXED_RUST";
+    pub const BLESS: &str = "RUSTFIX_TEST_BLESS";
```

Comment:
This makes me wonder if we should use `snapbox` to minimize the custom logic here.